ui/src/views/404/index.jsx

```diff
@@ -1,5 +1,52 @@
 // @flow
+import { colour } from 'pasteup';
+import Logo from 'assets/images/guardian-logo-320.svg';
+import Navigation from './Navigation';
+import Footer from './Footer';
 
-import style from './style.js.scss';
+import {
+    heading,
+    fluidWrap,
+    topbar,
+    subHeadingContainer,
+    subHeading,
+    para,
+} from './style.js.scss';
 
-export default () => <div style={style.message}>404</div>;
+export default () =>
+    <div style={fluidWrap}>
+        <div style={topbar}>
+            <a
+                href="/"
+                style={{
+                    color: '#005689',
+                    textDecoration: 'none',
+                    fontFamily: 'arial',
+                    padding: '8px 10px 7px',
+                    display: 'block',
+                }}>
+                Home
+            </a>
+        </div>
+        <Logo
```

Comment:
Had a chat with Simon about this on Friday and we both thought, it could maybe better to have a generic `SVG` component, which receives the filename as parameter. What do you think?